recipes/libftdi/1.x/conandata.yml

```diff
@@ -0,0 +1,16 @@
+sources:
+  "1.5":
+    sha256: 7c7091e9c86196148bd41177b4590dccb1510bfe6cea5bf7407ff194482eb049
+    url: "https://www.intra2net.com/en/developer/libftdi/download/libftdi1-1.5.tar.bz2"
+
+  "1.4":
+    sha256: ec36fb49080f834690c24008328a5ef42d3cf584ef4060f3a35aa4681cb31b74
+    url: "https://www.intra2net.com/en/developer/libftdi/download/libftdi1-1.4.tar.bz2"
+
+  "1.3":
+    url: "https://www.intra2net.com/en/developer/libftdi/download/libftdi1-1.3.tar.bz2"
+    sha256: 9a8c95c94bfbcf36584a0a58a6e2003d9b133213d9202b76aec76302ffaa81f4
+
+  "1.2":
+    url: "https://www.intra2net.com/en/developer/libftdi/download/libftdi1-1.2.tar.bz2"
+    sha256: a6ea795c829219015eb372b03008351cee3fb39f684bff3bf8a4620b558488d6
```

Comment:
Do we need all versions?
Isn't 1.5 enough?

I propose only adding 1.5.
If somebody really needs an older version, he/she can open a pr.